### PR TITLE
BaSP-TG-23: Added GetAll and Create Test  of Admin

### PR DESCRIPTION
**Pull request by NicolasGulam**
Added Tests for Admin: GetAll Employees and Create an Admin

### CODE REVIEW

File: src/controllers/admins.js [hunk 1]
```diff
@@ -67,15 +67,20 @@ const getAllAdmins = async (req, res) => {
...
     }
     return res.status(200).json({
       message: 'Admins found',
       data: admins,
+      error: false,
     });
   } catch (err) {
     return res.status(404).json({
       message: `An error ocurred: ${err}`,
+      data: undefined,
```
Comment (Licamus) [inline src/controllers/admins.js:82]: same as above

File: src/controllers/admins.js [hunk 2]
```diff
@@ -93,10 +98,13 @@ const createAdmin = async (req, res) => {
     return res.status(201).json({
       message: 'Admin created',
       data: result,
+      error: false,
     });
   } catch (err) {
     return res.status(400).json({
       message: `An error ocurred: ${err}`,
+      data: undefined,
```
Comment (Licamus) [inline src/controllers/admins.js:106]: same as above

File: src/controllers/admins.js [hunk 3]
```diff
@@ -67,15 +67,20 @@ const getAllAdmins = async (req, res) => {
...
+        error: true,
       });
     }
     return res.status(200).json({
       message: 'Admins found',
       data: admins,
+      error: false,
     });
   } catch (err) {
     return res.status(404).json({
```
Comment (Licamus) [inline src/controllers/admins.js:80]: Should be 400?

File: src/controllers/admins.js [hunk 4]
```diff
@@ -67,15 +67,20 @@ const getAllAdmins = async (req, res) => {
     if (!admins.length) {
       return res.status(404).json({
         message: 'Admin not found',
+        data: undefined,
```
[the remark this answers is not in the document]
Comment (ignacioaristo) [reply src/controllers/admins.js:70]: i agree

**Review by ignacioaristo — CHANGES_REQUESTED** [review]

File: src/controllers/admins.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (NicolasGulam) [reply src/controllers/admins.js]: Done

File: src/controllers/admins.js [hunk 5]
```diff
@@ -67,15 +67,19 @@ const getAllAdmins = async (req, res) => {
     if (!admins.length) {
       return res.status(404).json({
         message: 'Admin not found',
+
```
Comment (ignacioaristo) [inline src/controllers/admins.js:70]: remove this line

**Review by ignacioaristo — CHANGES_REQUESTED** [review]

File: src/controllers/admins.js (hunk 5 above)
Comment (NicolasGulam) [reply src/controllers/admins.js]: Done

**Review by ignacioaristo — APPROVED** [review]

**Pull request merged**